yarn-project/merkle-tree/src/indexed_tree/indexed_tree.ts

```diff
@@ -227,10 +258,18 @@ export class IndexedTree implements MerkleTree {
 
   /**
    * Saves the initial leaf to this object and saves it to a database.
+   * TODO: what will the size be
    */
-  private async init() {
+  private async init(initialSize = 0) {
```

Comment:
```suggestion
  private async init(initialSize = 1) {
```

I mean, initial size seems like it is never zero